app/src/main/res/values/strings.xml

```diff
@@ -181,6 +182,7 @@
   <string name="no">No</string>
   <string name="media_detail_caption">Caption</string>
   <string name="media_detail_title">Title</string>
+  <string name="media_detail_depiction">Depiction</string>
```

Comment:
Plural "Depictions", just like "Categories" is plural too.

Reply:
I'll add `Depictions` in the next comit